client/layout/masterbar/style.scss

```diff
@@ -306,9 +306,7 @@ body.is-mobile-app-view {
 		}
 
 		&.masterbar__reader {
-			.masterbar_svg-reader {
-				padding: 0;
-			}
+			padding: 0 9px 0 6px;
```

Comment:
Not a blocker, but due to the width of this icon and paddings I find the selected style a bit odd compared to the others around here, especially on the mobile bar. Its just so much wider than other selected items, it almost feels awkwardly wide.

<img width="234" alt="Screenshot 2024-08-22 at 12 25 43 PM" src="https://github.com/user-attachments/assets/883d01ef-2722-4666-a401-f9ba6a719733">
<img width="115" alt="Screenshot 2024-08-22 at 12 25 55 PM" src="https://github.com/user-attachments/assets/8cdd31fd-f43b-4769-91b6-2dbfd660125d">
<img width="126" alt="Screenshot 2024-08-22 at 12 25 59 PM" src="https://github.com/user-attachments/assets/f10e033f-1311-47a6-a662-cf2afd76f651">

Not necessarily suggesting any change. If the design side is happy with it, it seems fine to me.